lib/arc_ecto/schema.ex

```diff
@@ -31,6 +31,7 @@ defmodule Arc.Ecto.Schema do
           params
           |> Arc.Ecto.Schema.convert_params_to_binary
           |> Dict.take(allowed)
+          |> Enum.drop_while(fn({field, file}) -> is_nil(file) end)
```

Comment:
You want to use `Enum.reject/2` here, not `Enum.drop_while/2`. Can you update this?